PhotoViewer/button_commands.py

```diff
@@ -1,44 +1,120 @@
+from tkinter.colorchooser import askcolor
 from image_handler import ImageHandler
+import tkinter as tk
+from tkinter import PhotoImage
 
 imageObject = ImageHandler()
```

Comment:
You really did well the separation of concerns between your app and the image handler.
However, there is a slightly smarter way to manipulate this inside the Button Handler.

Why don't you inject the ImageHandler in the constructor instead of injecting the height, width etc... and then creating the handler internally.

Advantages:

Height and width are not used by the button handler. so they should not be injected.
If you change your image handler. You only inject it in the buttonshandler without changing any code internally...